index.js

```diff
@@ -1,17 +1,59 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    let ascNumbers = this.items.sort();
+    this.items = ascNumbers;
+    this.length = this.items.length;
+  }
 
-  get(pos) {}
+  get(pos) {
+    let element = this.items[pos];
 
-  max() {}
+    if(this.items[pos]) {
+      return element;
+    } else {
+      throw new Error('OutOfBounds');
+    }
+  }
 
-  min() {}
+  max() {
+    if(this.items) {
+      this.items.reduce((el1, el2) => {
+        let maxNumber = Math.max(el1, el2);
+        return maxNumber;
+      });
+    } else {
+      throw new Error('EmptySortedList');
+    }
+  }
 
-  sum() {}
+  min() {
+    if(this.items) {
+      this.items.reduce((el1, el2) => {
+        let minNumber = Math.min(el1, el2);
+        return minNumber;
+      });
+    } else {
+      throw new Error('EmptySortedList');
+    }
+  }
 
-  avg() {}
+  sum() {
+    if(this.items) {
+      return this.items.reduce((el1, el2) => el1 + el2);
+    } else {
+      return 0;
+    }
```

Comment:
Consider using the length of the array in your condition here instead.

```suggestion
    if(this.items.length) {
      return this.items.reduce((el1, el2) => el1 + el2);
    } else {
      return 0;
    }
```